h/accounts/models.py

```diff
@@ -130,6 +130,11 @@ def is_activated(self):
     _password = sa.Column('password', sa.UnicodeText(), nullable=False)
     # Password salt
     salt = sa.Column(sa.UnicodeText(), nullable=False)
+    # Last password update
+    last_password_update = sa.Column(sa.DateTime(timezone=False),
```

Comment:
`timezone=False` is [the default](http://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.DateTime) for `sa.DateTime`, so you can omit it here.